msteams-platform/webhooks-and-connectors/how-to/add-incoming-webhook.md

```diff
@@ -15,6 +15,9 @@ An Incoming Webhook lets external applications to share content in Microsoft Tea
 * Message extensions
 * Connectors
 
+> [!IMPORTANT]
+> Since Teams Toolkit is launched now, we recommend you to download [Teams Toolkit](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension) and build Teams Bot app which performs same behavior instead of creating Incoming Webhooks. You can start with [Build your first notification bot](https://docs.microsoft.com/en-us/microsoftteams/platform/sbs-gs-notificationbot) or [incoming webhook notification sample](https://github.com/OfficeDev/TeamsFx-Samples/tree/dev/incoming-webhook-notification). Learn more at [Teams Toolkit documents](https://docs.microsoft.com/en-us/microsoftteams/platform/toolkit/teams-toolkit-fundamentals).
```

Comment:
Instead of saying "You should use Teams toolkit because Teams toolkit is launched now", I'd like to change it to provide 2 points in this paragraph to help people to understand why to recommend it:
1. Other than incoming webhook, there is another option "notification bot" can do the similar but more powerful functionalities, and give the link of the table in https://review.docs.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/what-are-webhooks-and-connectors#notification-bot-or-incoming-webhook---choose-the-right-one
2. Teams toolkit can support both and easier way for you to get started, so please try to download it.

Reply:
thanks, have made changes accordingly.